hr_attendance_calendar/reports/extra_hours_evolution_day_report.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2018 Compassion CH (http://www.compassion.ch)
+#    @author: Samuel Fringeli <samuel.fringeli@me.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import tools
+from odoo import models, fields, api
+
+
+class ExtraHoursEvolutionDayReport(models.Model):
+    _name = "hr_attendance_day.extra_hours_evolution_day_report"
```

Comment:
I would suggest `hr.attendance.balance.evolution.report` as shorter name. The convention is to not use `_` in model names.